docker/bin/cluster_up.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash -eu
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+source ${DOCKER_BIN}/../.env
+
+# Wait for http service listener to come up and start serving
+# $1 http service name
+# $2 http service address
+await_http() {
+    local exit_code
+    local attempt=0
+
+    until [[ `curl ${2}` ]] || [[ ${attempt} -gt 5 ]]; do
```

Comment:
Are the backticks really needed here?
I think you should do: `until curl ${2} || [[ ${attempt} -gt 5 ]]; do`